src/main/msp/msp.c

```diff
@@ -1370,7 +1370,7 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, motorConfig()->dev.motorPwmProtocol);
         sbufWriteU16(dst, motorConfig()->dev.motorPwmRate);
         sbufWriteU16(dst, motorConfig()->digitalIdleOffsetValue);
-        sbufWriteU8(dst, gyroConfig()->gyro_use_32khz);
+        sbufWriteU8(dst, 0);
```

Comment:
Maybe is a good idea to add a comment in all the values "removed", in this way we know which was the legacy field sent here.

Reply:
Cool, handled in d629c9d - I put `DEPRECATED: <name_of_the_thing>`. is `DEPRECATED:` OK? or prefer `REMOVED:`?

Reply:
To me is right. DEPRECATED is the right word.

Maybe is better to add the comment in line after the sentence and not upper to it, but this is only my preference 😋

Reply:
I can do that. 👍